ios/FluentUI/Other Cells/ActionsCell.swift

```diff
@@ -95,9 +129,9 @@ open class ActionsCell: UITableViewCell {
     private var action1Type: ActionType = .regular
     private var action2Type: ActionType = .regular
 
-    private let topSeparator = Separator(style: .default, orientation: .horizontal)
-    private let bottomSeparator = Separator(style: .default, orientation: .horizontal)
-    private let verticalSeparator = Separator(style: .default, orientation: .vertical)
+    private let topSeparator = MSFDivider()
```

Comment:
There seems to be a cocoapods issue with not being able to find MSFDivider

Reply:
Yes. Just noticed it. Validating locally and will push the fix for it soon.
There is no subspec for the divider and it should be a dependency for the TableViewCell.